doc/make.py

```diff
@@ -1,228 +0,0 @@
-#!/usr/bin/env python
```

Comment:
OK, this all looks great to me, and it works, and the tests pass, so I think its good to go.  *But* do we need to remove `make.py`?  Someone may have a fancy script that uses it for something and want to keep it around, maybe with a big warning at the beginning that its going away soon?  

The converse of that argument is that it might be confusing to have both.  

I don't really care, but was just trying to think of what could go wrong.

Reply:
I don't think we ever made any guarantees as to backcompatibility of tooling (and wouldn't want to do so either).

Reply:
Even I think we don't need to worry about this too much.  The downstream packagers (debian, fedora, etc) will need to adapt their build scripts, but it is clearly documented what to change it too.